ckanapi/bld.bat

```diff
@@ -0,0 +1,8 @@
+"%PYTHON%" setup.py install
```

Comment:
When a package uses `setuptools` we can add `--single-version-externally-managed --record record.txt` here and below.  That makes the package more "portable."